src/database/models/add_comment.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Comments = sequelize.define('Comments', {
+    comment: { type: DataTypes.STRING, allowNull: false },
+  }, { freezeTableName: true });
```

Comment:
There should be a functionalities for `replying to the comment`
lihe this:
```
parentCommentId: { type: DataTypes.Integer, allownull: true },
```
the, if `parentCommentId` is `null` will be able to identify that the comment was not a reply